app/components/grouped_cards/card_component.rb

```diff
@@ -0,0 +1,21 @@
+module GroupedCards
+  class CardComponent < ViewComponent::Base
+    attr_reader :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    def header
+      @data["header"]
+    end
+
+    def fields
+      @data.without("header")
+    end
+
+    def linked_value(value)
+      Rinku.auto_link(h(value)).html_safe
```

Comment:
We have a helper you could use instead of `html_safe`, but since you're escaping anyway take it or leave it:

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/helpers/text_formatting_helper.rb#L10

Reply:
Thanks, thats got different behaviour though so I'll leave it as is - that would still allow HTML through which I'm not wanting to - Rinku doesnt handle escaping well - hence the above approach (I really don't like using `html_safe` though)